.github/manual-test

```diff
@@ -48,14 +49,102 @@ fi
 
 printf "\nDid everything work as expected? [yN] "
 read -r response
+
+if [ -n "$tmp_dir" ]; then
+  printf "\n***** Deleting test dir %s *****\n" "$tmp_dir"
+  rm -rf "$tmp_dir"
+  printf "\n"
+fi
+
+if [ "$response" != 'y' ]; then
+  echo "Back to fixing"
+  exit 1
+else
+  echo "Yayyyyyyyyyyy"
+fi
+
+printf "\n***** Creating test dirs & resources *****\n"
+tmp_dir=$(mktemp -d -t nativefier-manual-test-auth-XXXXX)
+
+printf "\n***** Building test app *****\n"
+node ./lib/cli.js 'http://httpbin.org/basic-auth/foo/bar' \
+  --basic-auth-username foo \
+  --basic-auth-password bar \
+  --name "app" \
+  "$tmp_dir"
+
+printf "\n***** Test checklist *****
+- Was successfully logged in via HTTP Basic Auth. Should see:
+    {
+      \"authenticated\": true, 
+      \"user\": \"foo\"
+    }
+- Console: no Electron runtime deprecation warnings/error logged
+"
+
+printf "\n***** Running app *****\n"
+if [ "$(uname -s)" = "Darwin" ]; then
+  open -a "$tmp_dir/app-darwin-x64/app.app"
+else
+  "$tmp_dir/app-linux-x64/app"
+fi
```

Comment:
Pls extract into a lil' `launch_app` helper function defined at the beginning of the script